docs/addons/index.md

```diff
@@ -8,28 +8,29 @@ Within the context of the `ssp-amazon-eks` framework, an add-on is abstracted as
 
 The framework currently supports the following add-ons.
 
-| Add-on             | Description                                                                       |
-|-------------------|-----------------------------------------------------------------------------------|
-| [`AppMeshAddOn`](./app-mesh) | Adds an AppMesh controller and CRDs (pending validation on the latest version of CDK) |
-| [`ArgoCDAddOn`](./argo-cd) | Provisions Argo CD into your cluster. |
-| [`AWS Load Balancer Controller`](./aws-load-balancer-controller) | Provisions the AWS Load Balancer Controller into your cluster. |
-| [`CalicoAddOn`](./calico) | Adds the Calico 1.7.1 CNI/Network policy engine |
-| [`ClusterAutoscalerAddOn`](./cluster-autoscaler) | Adds the standard cluster autoscaler |
-| [`ContainerInsightsAddOn`](./container-insights) | Adds Container Insights support integrating monitoring with CloudWatch |
-| [`CoreDnsAddOn`](./coredns.md) | Adds CoreDNS Amazon EKS add-on. CoreDNS is a flexible, extensible DNS server that can serve as the Kubernetes cluster DNS |
-| [`KubecostAddOn`](./kubecost.md) | Adds [Kubecost](https://kubecost.com) cost analyzer to the EKS cluster |
-| [`KarpenterAddOn`](./karpenter.md) | Adds [Karpenter](https://github.com/awslabs/karpenter) support for Amazon EKS. | 
-| [`KubeProxyAddOn`](./kube-proxy.md) | Adds kube-proxy Amazon EKS add-on. Kube-proxy maintains network rules on each Amazon EC2 node |
-| [`MetricsServerAddOn`](./metrics-server.md) | Adds metrics server (pre-req for HPA and other monitoring tools)|
-| [`ExternalDnsAddOn`](./external-dns) | Adds [External DNS](https://github.com/kubernetes-sigs/external-dns) support for AWS to the cluster, integrating with Amazon Route 53
-| [`MetricsServerAddOn`](./metrics-server) | Adds metrics server (pre-req for HPA and other monitoring tools)|
-| [`NginxAddOn`](./nginx.md) | Adds NGINX ingress controller |
-| [`SecretsStoreAddOn`](./secrets-store.md) |  Adds AWS Secrets Manager and Config Provider for Secret Store CSI Driver to the EKS Cluster |
-| [`SSMAgentAddOn`](./ssm-agent.md) | Adds [Amazon SSM Agent](https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html) to worker nodes |
-| [`VpcCniAddOn`](./vpc-cni.md) | Adds the Amazon VPC CNI Amazon EKS addon to support native VPC networking for Amazon EKS |
-| [`Weave GitOps`](https://github.com/weaveworks/weave-gitops-ssp-addon) | [Weave GitOps Core](https://www.weave.works/product/gitops-core/) AddOn |
-| [`XrayAddOn`](./xray) | Adds XRay Daemon to the EKS Cluster |
-| [`VeleroAddOn`](./velero.md) | Adds [Velero](https://velero.io/) to the EKS Cluster |
+| Add-on                                                                 | Description                                                                                                                           |
```

Comment:
Is there a formatting issue here? Why all of the entries highlighted as change? It makes it harder to figure out what changed specifically.

Reply:
It is a formatting in the original document. According to the IDE, there are potential rendering issues with markdown tables when the columns do not align properly. As such the table was reformatted. The only new entry for this add-on is :

| [`EbsCsiDriverAddOn`](./ebs-csi-driver.md)                             | Adds EBS CSI Driver Amazon EKS add-on. This driver manages the lifecycle of Amazon EBS volumes for persistent storage                 |

Reply:
It was clear to me that IDE reformatted, but we did not have rendering issues. And, yes, I surely saw the add-on. I think we should turn off automatic reformat, and only apply it when the intention is to reformat. 
it is minor.